modules/ztp-image-based-upgrade-prep-resources.adoc

```diff
@@ -19,9 +27,9 @@ The {lcao} needs all your OADP resources wrapped in a `ConfigMap` objects to pro
 
 .Procedure
 
-. Ensure that you created the `openshift-adp` and `ztp-common` namespaces.
-The root policy is created in the `ztp-common` namespace.
-If you use another namespace, update the `example-oadp-policy.yaml` policy accordingly.
+. Ensure that you created the `openshift-adp` and `ztp-group` namespaces.
```

Comment:
Procedure 1 not necessary 
Procedure 4 is redundant. It’s covered in P2. 
Procedure 3 for application backup and restore CR should be the examples from “Doc additions”

The whole content in kustomization.yaml is needed for generating oadp cm in target cluster, so the procedure is something like

1.  create oadp-cm configmap through oadp-cm-policy  in a new PGT ibu-upgrade-ranGen.yaml
 ```
     ...
      sourceFiles:
      - fileName: ConfigMapGeneric.yaml
        policyName: "oadp-cm-policy"
        metadata:
          name: oadp-cm
          namespace: openshift-adp
```
2.  add the kustomization.yaml with the following content
```
apiVersion: kustomize.config.k8s.io/v1beta1
kind: Kustomization

generators: <1>
- ibu-upgrade-ranGen.yaml

configMapGenerator: <2>
- files:
  - source-crs/ibu/PlatformBackupRestore.yaml
  # - <add-workload-oadp-crs-here>
  name: oadp-cm
  namespace: ztp-group
generatorOptions:
  disableNameSuffixHash: true 


patches: <3>
- target:
    group: policy.open-cluster-management.io
    version: v1
    kind: Policy
    name: group-ibu-oadp-cm-policy
  patch: |-
    - op: replace
      path: /spec/policy-templates/0/objectDefinition/spec/object-templates/0/objectDefinition/data
      value: '{{hub copyConfigMapData "ztp-group" "oadp-cm" hub}}'
```
<1> generates oadp cm policy
<2> create a configmap called oadp-cm on the hub cluster with backup and restore CRs
<3> patch to override the data field of configmap added in oadp-cm-policy with hub template to propogate the oadp-cm configmap to all target clusters
3. push the change to git and etc